lib/fathead/node_js/parse.js

```diff
@@ -30,12 +31,13 @@ function createItem(ops) {
 }
 
 function getAbstract(obj) {
-    var a = obj.desc ? '<p id="obj.desc">' + obj.desc + '</p>' : '',
+    var summarizedDesc = obj.desc ? preProcessDescription(obj.desc) : '';
+    var a =  summarizedDesc,
         code = '',
-        a = a.replace(/<\s*p*>/ig, ''),
+     // a = a.replace(/<\s*p*>/i, ''),
```

Comment:
@ahetawal-p Looks good, but we should remove any commented out code from the PR (we can always retrieve it through Git). Cheers!